README.md

```diff
@@ -120,20 +126,29 @@ data layouts based on Zipkin's V1 Thrift model, as opposed to the
 simpler v2 data model currently used.
 
 #### MySQL
-The [MySQL v1](zipkin-storage/mysql-v1) component currently is only
-tested with MySQL 5.6-7. It is designed to be easy to understand, and
-get started with. For example, it deconstructs spans into columns, so
+The [MySQL v1](zipkin-storage/mysql-v1) component uses MySQL 5.6+
+features, but is tested against MariaDB 10.3.
+
+The schema was designed to be easy to understand and get started with;
+it was not designed for performance. Ex spans fields are columns, so
 you can perform ad-hoc queries using SQL. However, this component has
 [known performance issues](https://github.com/apache/incubator-zipkin/issues/1233): queries will eventually take seconds to return
 if you put a lot of data into it.
 
+This store does not require a [job to aggregate](https://github.com/apache/incubator-zipkin-dependencies) dependency links.
+However, running the job will improve performance.
```

Comment:
It might be good to clarify that the performance improvement is only for retrieving dependency links - that is, it won't improve overall performance of read/write on span data.

Reply:
great point